### PR TITLE
feat(starr-anime): Remove unnecessary anime remux groups, move top remux groups to higher tiers, add NAN0 and VULCAN

### PR DESCRIPTION
**Pull request by rg9400**
# Pull Request

## Purpose

Removing a variety of anime remux groups that do not have a single release marked as Best on SeaDex. Adding VULCAN to the remux tier since they have a few best releases. Moving ZR to tier 3 because of huge number of best releases. Moving Dragon-Releases to tier 4 for having 10+ best releases. Adding NAN0 to tier 4 for being a remux group with 10+ best releases. TTGA has over 50 best releases, but they are almost exclusively when the alternative is WEB only. For now, we can keep them in tier 5 so they get trumped by better encodes.

Count per Ravencentric:
```
{'ANThELIa': 0, 'BluDragon': 4, 'D4C': 5, 'Dragon-Releases': 11, 'E.N.D': 1, 'KAWAiREMUX': 0, 'MKVULTRA': 0, 'Raizel': 2, 'REVO': 9, 'Spark': 0, 'SRLS': 2, 'ZR': 0, '-ZR-': 151, 'NAN0': 15, 'VULCAN': 3, 'FraMeSToR': 8, 'TTGA': 49}
```

## Approach

I reordered Tier 3 in both Radarr and Sonarr because they were not alphabetical. Otherwise, standard regex. I did add an extra condition for ZR to capture `-ZR-`

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/json/sonarr/cf/anime-bd-tier-04-seadex-muxers.json [hunk 1]
```diff
@@ -250,175 +250,195 @@
...
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": false,
+      "fields": {
+        "value": "\\b(xPearse)\\b"
       }
     }
   ]
+}
+  ]
```
Comment (TRaSH-) [inline docs/json/sonarr/cf/anime-bd-tier-04-seadex-muxers.json:443]:
```suggestion
 
```

Comment (TRaSH-) [inline docs/json/sonarr/cf/anime-bd-tier-04-seadex-muxers.json:443]:
```suggestion
```

**Review by TRaSH- — CHANGES_REQUESTED** [review]
These two lines needs to be removed

**Review by TRaSH- — APPROVED** [review]

**Review by FonduemangVI — APPROVED** [review]
Good to go

**Pull request merged**